synapse/handlers/presence.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, hs: "HomeServer"):
 
     @abc.abstractmethod
     async def user_syncing(
-        self, user_id: str, affect_presence: bool
+        self, user_id: str, affect_presence: bool, presence_state: str
```

Comment:
The docstring below could use an update now that there's a new argument.
The same for the rest of the argument changes in this PR.

Reply:
Done (where the was docstring for any of the args)

Reply:
Thanks for keeping the docstrings updated!